quarkus-solace-client/deployment/src/main/java/com/solace/quarkus/deployment/DevServicesSolaceProcessor.java

```diff
@@ -245,6 +242,11 @@ protected void configure() {
             }
         }
 
+        @Override
+        protected void containerIsStarted(InspectContainerResponse containerInfo) {
+            executeCommand("chown 1000:0 -R /var/lib/solace");
```

Comment:
I am just going through logs before merging it looks like this command fails. In the docs it mentions that we need to set the permission to source folder. I think we can remove it

<img width="1254" alt="image" src="https://github.com/SolaceLabs/solace-quarkus/assets/83568543/bddf9e5d-dcc7-4a7f-a697-2408c59048f6">

Reply:
I did a quick run on my local by enabling podman, and i didnt see any major issue while using dev container. Let's merge this now and if any unexpected issue crops up will address in next version.